src/shared/utils.ts

```diff
@@ -110,4 +111,11 @@ export class Utils {
   
     return metaData;
   }
+
+  public static interpolateFile(sls: Serverless, path: string, params: Map<string, string>) {
+    const template = sls.utils.readFileSync(path);
+    const names = params.keys();
+    const vals = params.values();
+    return new Function(...names, `return \`${template}\`;`)(...vals);
```

Comment:
Q: can 
```
`return \`${template}\`;`
```
be replace with 
```
`return ${template};`
```

Reply:
This is actually a string interpolation that won't happen here. We actually have the same function (kind of) in VoTT. This is expecting a file with templated variables. I'm actually not using this right now... I was using it when I had a javascript template file. But I might keep it in just in case we go back to that